wdl-lint/tests/lints/blank-lines-between-elements/source.wdl

```diff
@@ -0,0 +1,92 @@
+#@ except: DescriptionMissing, ImportWhitespace, InputSorting, LineWidth, MissingMetas, MissingOutput, MissingRuntime, Whitespace
```

Comment:
That would be a welcome change to the testing framework. I would much rather specify which rule is being tested and ignore everything else.

Reply:
I misunderstood you at first. I've reenabled the other whitespace rules.

Reply:
Not a change for this PR, but IMO it's problematic how long our `#@ except` comments have gotten for the tests. I think the right long term solution is to make our test cases lint-compliant rather than except a whole big list of lints for every test case.

The tests should show us what a user is going to see, shouldn't it?

Reply:
(and we certainly hope that users aren't using big lists of `#@ except` comments)

Reply:
> Not a change for this PR, but IMO it's problematic how long our `#@ except` comments have gotten for the tests. I think the right long term solution is to make our test cases lint-compliant rather than except a whole big list of lints for every test case.
> 
> The tests should show us what a user is going to see, shouldn't it?

You're probably right. Writing fully compliant WDL is a little overboard depending on the test though.

Reply:
That's true as well. There's a happy medium to be found. Right now I'd argue our tests ignore/except too much.